src/components/CTable.vue

```diff
@@ -70,18 +93,35 @@ export default {
           .includes(this.filter.toLowerCase())
       );
     }
+  },
+  methods: {
+    isSortable(column) {
+      return Object.prototype.hasOwnProperty.call(column, 'sortable') && column.sortable;
+    },
+    sortColumn(columnName) {
+      if (columnName === this.currentColumnSorted) {
+        this.directionColumnSorted = this.directionColumnSorted === 'up' ? 'down' : 'up';
+      } else {
+        this.currentColumnSorted = columnName;
+        this.directionColumnSorted = 'up';
+      }
+      this.messages.sort((row1, row2) => {
```

Comment:
Sort function in js is dumb but from the doc to compare number (or date) you can use subtraction: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort
```
this.messages.sort((row1, row2) => {
        const direction = this.directionColumnSorted === 'asc' ? 1 : -1;
        return direction * (row1[columnName] - row2[columnName]);
      });
```